packages/parser/src/fs.ts

```diff
@@ -110,8 +129,8 @@ export async function load(userRoot: string, filepath: string, content?: string,
   }
 }
 
-export async function save(markdown: SlidevMarkdown) {
+export function save(markdown: SlidevMarkdown) {
```

Comment:
I would prefer to keep this function async, in case we need async operation in the future.